Kickstarter-iOS/Features/PledgePaymentMethods/Controller/PledgePaymentMethodsViewController.swift

```diff
@@ -195,7 +212,7 @@ final class PledgePaymentMethodsViewController: UIViewController {
           label: existingPaymentOption.label
         )
         strongSelf.viewModel.inputs
-          .paymentSheetDidAdd(newCard: paymentDisplayData, setupIntent: clientSecret)
+          .paymentSheetDidAdd(newCard: paymentDisplayData, clientSecret: clientSecret)
```

Comment:
I've been trying to rename `setupIntent` to `clientSecret` or `stripeIntent` wherever I can, but definitely have missed some spots.